src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -289,6 +308,45 @@ def _extract_message_id(response_body: bytes) -> Optional[str]:
             )
         )
 
+    @staticmethod
+    def _extract_queue_url(request_body: bytes) -> Optional[str]:
+        return safe_key_from_query(request_body, "QueueUrl")
+
+
+class SqsJsonParser(BaseSqsParser):
+    @staticmethod
+    def _extract_message_id(response_body: bytes) -> Optional[str]:
+        parsed_body = {}
+        try:
+            parsed_body = json.loads(response_body)
+        except json.JSONDecodeError as e:
+            get_logger().warning("Error while trying to parse sqs json request body", exc_info=e)
+
+        # If the request was to send a single message this should work
+        message_id = parsed_body.get("MessageId")
+        if message_id and isinstance(message_id, str):
+            return message_id
+
+        # This should work if the request was to send a batch of messages.
+        # Find the message id of the first successful message, and if there are none the first failed message
+        messages = parsed_body.get("Successful", []) + parsed_body.get("Failed", [])
```

Comment:
If a message has failed to be sent, why do we need its messageId? 🤔

Reply:
This is the current implementation for SQS xml parsing so I copied it.

BTW it doesn't hurt to add the (true I might add) piece of information to the parsing of the request, it could be used in a further part of the pipeline if required

Reply:
1. the current implementation does not have failed messages 🤔 
2. adding unused information is wasteful

Reply:
My bad, I thought the current implementation did do this.
Removed the failed message id